src/index.js

```diff
@@ -0,0 +1,280 @@
+import './style.css';
+import getData from './modules/getData.js';
+import getLikes from './modules/getlikes.js';
+import likesShow from './modules/likeShow.js';
+import createAppId from './modules/createAppId.js';
+import postLikes from './modules/postLikes.js';
+import showReservation from './modules/showReservation.js';
+import createReservation from './modules/createReservation.js';
+import showComments from './modules/showComments.js';
+
+const mainItemsContainer = document.querySelector('.main_items_container');
+let comments;
+let reservation;
+const backdrop = document.querySelector('.backdrop');
+const projectID = 'jz9Xjlf6GBUQcvBtrKpI';
+const dateTime = new Date();
+const navContainer = document.querySelector('.nav_container');
+
+const closePopup = () => {
+  backdrop.innerHTML = '';
+  backdrop.classList.add('hidden');
+};
+
+let commentHtml = '';
+let commentsCounter = 0;
+let reservationCounter = 0;
+
+const commentsShow = (data) => {
+  commentHtml = '';
+  commentsCounter = 0;
+  if (!data) return;
+  data.forEach((d) => {
+    commentHtml += `<p>${d.creation_date} ${d.username}: ${d.comment}</p>`;
+    commentsCounter += 1;
+  });
+};
+
+const reservationShow = (data) => {
+  commentHtml = '';
+  reservationCounter = 0;
+  if (!data) return;
+  data.forEach((d) => {
+    commentHtml += `<p>${d.date_start} - ${d.date_end} by ${d.username}</p>`;
+    reservationCounter += 1;
+  });
+};
```

Comment:
- [ ] Well done implementing the reservation counter as suggested by the previous reviewer, but there seems to be the same small issue as it was in the comments counter. In the requirements, it says that the screen should get updated when the user clicks the button. now when the user clicks the button, the reservation appears but the counter doesn't get updated. it gets updated when you click to open the popup the next time. the reason for that is you are calling this reservationShow function in the createPopupWindowReservation function so reservationShow only gets called when createPopupWindowReservation gets called. kindly consider fixing this small issue. I think that the counter should get updated right after the user clicks on the reserve button.
- When the user clicks on the "Reserve" button, the data is recorded in the Involvement API and the screen is updated.

https://user-images.githubusercontent.com/96683843/234917986-b47b0f44-96f4-45f9-9601-0268d08db915.mov

Reply:
Implemented ✅✅. Thanks for the suggestions✌️.